engines/tinsel/dialogs.cpp

```diff
@@ -5211,6 +5221,28 @@ void Dialogs::idec_inv2(SCNHANDLE text, int MaxContents,
 	         100, 100, true);
 }
 
+/**
+ * Called from Glitter functions: dec_invMain
+ * - Declare inventories 1,3 and 4 and hail the loadingscreen(?) scene.
+ */
+void Dialogs::idec_invMain(SCNHANDLE text, int MaxContents) {
+	idec_inv(INV_1, text, MaxContents,3, 2, 3, 2, 3, 2, 39,
+			72, false);
+	idec_inv(INV_MENU, 0, 3, 2, 2, 2, 1, 3, 1, 100, 100,
+			false);
+	idec_inv(INV_4, text, MaxContents,3, 2, 3, 2, 3, 2, 39,
+			72, false);
+
+	warning("TODO: idec_invMain: implement language scene playback");
+	// This is not yet actived because playing the loadsceen scene
```

Comment:
I tested a bit, and the problem there is that INVPLAY (opcode 90) is not yet mapped, I have local changes that fix that.

Reply:
I originally stubbed it by mapping it to ZZZZZZ and then ran into the coroutine stack problem. I decided against including that in the commit because it does not add anything.

Reply:
Shall I add the stub to this commit?